tests/contrib/scrapy/test_pipeline_cerberus.py

```diff
@@ -135,4 +143,58 @@ def test_validation_from_url(mocker):
     pipe.process_item(test_item, None)
 
     assert "spidermon/validation/items/errors" in pipe.stats.stats.get_stats()
-    assert pipe.stats.stats.get_stats().get("spidermon/validation/validators/testitem/cerberus", True)
+    assert pipe.stats.stats.get_stats().get(
+        "spidermon/validation/validators/testitem/cerberus", True
+    )
+
+
+@pytest.mark.skipif(
+    sys.version_info < (3, 4), reason="mock requires python3.4 or higher"
+)
+def test_pipelines_with_cerberus_mocked_true_case(mocker):
+    mock_cerberus = mock.patch("spidermon.contrib.scrapy.pipelines.CerberusValidator")
+
+    class Cerberus(object):
+        def validate(self):
+            return (False, {})
+
+    mock_cerberus.return_value = Cerberus()
+    settings = {SETTING_CERBERUS: [cerberus_test_schema]}
+    test_item = TestItem({"url": "https://github.com"})
+    crawler = get_crawler(settings_dict=settings)
+    pipe = ItemValidationPipeline.from_crawler(crawler)
+    pipe.process_item(test_item, None)
+
+    assert STATS_ITEM_ERRORS not in pipe.stats.stats.get_stats()
+    assert pipe.stats.stats.get_stats().get(
```

Comment:
This is not good.
Why is the default value `True`?
If the pipeline is required to fill this stats, and it does not, this test will pass, which is a problem

Reply:
Oh, I was just testing this bit. I must have forgotten to change this. this actually defaults to false.

Reply:
Fixed.